Makefile

```diff
@@ -40,18 +44,22 @@ deps:
 	go get -u github.com/goadesign/gorma
 
 generate: $(DESIGNS)
-	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
-	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
-	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
-	go-bindata-assetfs -debug assets/...
-	godep get
+	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
```

Comment:
Even with multiple workspaces, multiple versions, you would allow the env tool' to control that, not our Makefile. e.g.  https://github.com/moovweb/gvm